1주차/1051.py

```diff
@@ -0,0 +1,14 @@
+n, m = map(int, input().split())
+arr = []
+
+for i in range(n):
+    arr.append(input())
```

Comment:
이거는 아래처럼 간단하게 변경할 수 있어요
```suggestion
arr = [input() for i in range(n)]
```